### PR TITLE
Added mobile requirements

### PR DESCRIPTION
**Pull request by jdiehl**
@ConnorLand

### CODE REVIEW

File: content/en/docs/refguide/mobile/getting-started-with-mobile/prerequisites.md [hunk 1]
```diff
@@ -11,11 +11,50 @@ aliases:
...
+### Progressive Web Apps
+
+There are no special requirements for developing, deploying, and accessing Progressive Web Apps (PWAs) with Mendix.
+
+### Native Mobile Apps
+
+Native mobile apps are not run in the device browser like web-based Mendix apps. Instead, a native app is built for each mobile operating system it should run on resulting in an APK or IPA file, which are then installed on the device. This introduces several special considerations:
+
+* **Developer Preview**: When developing a native mobile app, a special app called [Make It Native](/refguide/mobile/getting-started-with-mobile/prerequisites/) is required to preview the app. It is not possible to preview it in the browser or directly in Studio Pro using Design Mode.
+* **Build Native App**: For building native apps, we provide a tool called "Build Native App". This tool must be configured and run each time a new build is created. Build Native App is not supported on Studio Pro for Mac. Build Native App updates itself automatically by downloading a new version from AWS.
```
Comment (ConnorLand) [inline content/en/docs/refguide/mobile/getting-started-with-mobile/prerequisites.md]: Do I need to help make documentation for this tool so we can link to it?

Comment (jdiehl) [reply content/en/docs/refguide/mobile/getting-started-with-mobile/prerequisites.md]: This is not a new tool - let's not introduce too many changes in this PR

Comment (ConnorLand) [reply content/en/docs/refguide/mobile/getting-started-with-mobile/prerequisites.md]: I'll look into if Build Native App needs some minimal documentation and add it in a subsequent PR.

File: content/en/docs/refguide/mobile/getting-started-with-mobile/prerequisites.md [hunk 2]
```diff
@@ -11,11 +11,50 @@ aliases:
...
+
+Developing and building native mobile apps requires access to several online resources.
+
+For development:
+
+* To preview their apps, developers need to be able to connect their mobile device running Make It Native to their Laptop running Studio Pro on ports 8080 and 8083.
+
+For building:
+
+* Download updates from AWS: `https://appdev-mx-cdn.s3.amazonaws.com/native-builders/latest.exe`
```
Comment (ConnorLand) [inline content/en/docs/refguide/mobile/getting-started-with-mobile/prerequisites.md:48]: Using code tics makes the URLs easy to see on the docs site, but it renders them nonclickable. Is this the desired behavior, or do you want them clickable?

Comment (jdiehl) [reply content/en/docs/refguide/mobile/getting-started-with-mobile/prerequisites.md:48]: These links are for reference only and should not be clicked

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**